### ISSUE TITLE
Not playing well with Ableton provided OSC receiver.

### ISSUE THREAD

**Issue by jpnielsen** [j4s0n-c/trowaSoft-VCV#68]
Howdy, 

I'm attempting to use cvOSCcv to control some VST's in ableton.

I found this from Ableton, that takes input from a TouchOSC app, and lets me map the OSC values to ableton controllers.
https://github.com/Ableton/m4l-connection-kit/tree/master/OSC%20TouchOSC

It doesn't perform well, the update frequency is very slow.

<img width="1920" alt="image" src="https://user-images.githubusercontent.com/27916597/220480976-31c1f8c8-7c84-4b66-97d5-7131ae558132.png">

I'm thinking perhaps it has to do with the format that cvOSCcv sends, which I'm guessing is different from TouchOSC - perhaps the packing of the data is not understood by the max patch, it only uses a somple "udpreceive".

I don't know if you use Ableton and/or have a better max patch to work with OSC from cvOSCcv ?

Perhaps you can use the information, and make a "TouchOSC" mode, that will make it work with apps that accept touchOSC OSC messages (Which I'm guessing are simpler, perhaps only one controller per message).

Thank you.

**Comment by jpnielsen** [comment]
BTW, the max device works fine with TouchOSC

<img width="1920" alt="image" src="https://user-images.githubusercontent.com/27916597/220505103-8ef04d31-a710-4b10-9902-a4a3893b4b7c.png">


Attaching my modified amxd with 24 OSC receives.
And the touchosc panel.
[WaveState_remote.zip](https://github.com/j4s0n-c/trowaSoft-VCV/files/10799478/WaveState_remote.zip)

**Comment by j4s0n-c** [comment]
"It doesn't perform well, the update frequency is very slow."
I do not have live suite to test this...
The "speedlim 10" node in the max patch is slowing down the OSC to update only every 10ms. We are already limiting the sending speed on cvOSCcv to 100hz (by default). The two limiters have no way to synchronize so, you are probably only receiving OSC when they line up. Please try removing/bypassing the "speedlim 10" node in the max patch (connect output of "udpreceive 7400" to input of "p CheckDimensions" and let me know how it goes.

**Comment by jpnielsen** [comment]
I bypassed "speedlim 10", but still only updated once in a while.

Then i remembered preset .. copy / preset .. paste.


I changed
  "channelSensitivity": 0.05000000074505806
to 
  "channelSensitivity": 0.005000000074505806

and pasted the config to the cvOSCcv module.

Now, the changes per 1/100s are large enough to trigger an update, and the sliders move smooth in Ableton

Perhaps make the "channelSensitivity" configurable in the GUI. 

Thanks !

Note to fellow explorers .. the cvOSCcv expander modules have their own presets, that needs changing of the sensitivity too to get smooth updates.
